extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -219,36 +219,47 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
     },
 
     store: {
-      dicom: async dataset => {
+      dicom: async (dataset, uploadCallbacks = {}, abortSignal) => {
```

Comment:
`abortSignal` should have default if it is coming after the second one that has a default